src/app/questions/oneof/oneof.component.scss

```diff
@@ -0,0 +1,7 @@
+button:disabled {
+    color: rgba(0, 0, 0, 0.3) !important;
```

Comment:
do we need to specify this? or can we use the default styling of disabled

Reply:
we can use default styling, this part need to remove. i add this for testing. i will remove a push changes.